src/target/drivers/mcu/emu/protospi.h

```diff
@@ -3,8 +3,9 @@
 
 u8 PROTOSPI_read3wire();
 u8 PROTOSPI_xfer(u8 byte);
-#define PROTOSPI_pin_set(io) if(0) {}
-#define PROTOSPI_pin_clear(io) if(0) {}
+#define PROTOSPI_pin_set(io) if (0) {}
```

Comment:
I suggest to change to do {} while(0) so that if we missed a ; after the function, the compiler will notice it.